cumulusci/tasks/salesforce/update_dependencies.py

```diff
@@ -50,6 +50,15 @@ class UpdateDependencies(BaseSalesforceMetadataApiTask):
         "security_type": {
             "description": "Which users to install packages for (FULL = all users, NONE = admins only)"
         },
+        "prefer_2gp_from_release_branch": {
+            "description": "If True and this build is on a release branch (feature/NNN, where NNN is an integer), "
+            "or a child branch of a release branch, resolve GitHub managed package dependencies to 2GP builds present on "
+            "a matching release branch on the dependency."
+        },
+        "2gp_context": {
```

Comment:
Should we make this a property of the project__git section of the project config?

Reply:
I like this idea quite a bit so long as we also default it to "Build Feature Test Package" - I don't want to have to merge a PR in every single project that uses 2GP to make that change.

Reply:
Yeah, that's what I had in mind. Since we're inventing this convention for locating corresponding package versions, we can decide what to use as the default context for the commit status.

Possibly we should consider renaming it to something like "2gp Managed Beta Package Version" since it's no longer just for the feature test use case.